backend/config.ts

```diff
@@ -4,13 +4,13 @@ const toNumber = (value: string | undefined): number | undefined => {
 }
 
 export const app = {
-  port: process.env.PORT || 3002,
+  port: process.env.PORT || 3000,
   production: process.env.NODE_ENV === 'production',
   disableWebpack: process.env.DISABLE_WEBPACK === 'true',
   licencesUrl: process.env.LICENCES_URL || 'http://localhost:3003/',
   mailTo: process.env.MAIL_TO || 'feedback@digital.justice.gov.uk',
   tokenRefreshThresholdSeconds: toNumber(process.env.TOKEN_REFRESH_THRESHOLD_SECONDS) || 60,
-  url: process.env.PRISON_STAFF_HUB_UI_URL || `http://localhost:${process.env.PORT || 3002}/`,
+  url: process.env.PRISON_STAFF_HUB_UI_URL || `http://localhost:${process.env.PORT || 3000}/`,
```

Comment:
I can't remember if it was me that changed this - can you remember why the default port is changing here?

Reply:
Maybe would be good to change it back if we don't have a particular reason for change - who knows if this breaks other people's set ups etc!

Reply:
Thanks for taking a look. Good spot Jon, I reverted it back